app/build.gradle

```diff
@@ -2,6 +2,7 @@ plugins {
     id 'com.android.application'
     id 'kotlin-android'
     id 'com.google.gms.google-services'
+    id "org.sonarqube" version "3.5.0.2730"
```

Comment:
This change is not needed, it was for code analysis

Reply:
Oh neat.  SonarQube?  How did that work for you?

Reply:
It took a while to get working, but It gives some pretty good suggestions once it's working! It DID take me an hour to get it working and it broke after I made some changes, but it's pretty good when it's working.